docs/user/hosting.rst

```diff
@@ -1,28 +1,71 @@
-Documentation Hosting Features
-==============================
+.. old label
+.. _Documentation Hosting Features:
 
-The main way that users interact with your documentation is via the hosted HTML that we serve.
-We support a number of important features that you would expect for a documentation host.
+Hosting features provided by Read the Docs
+==========================================
 
-.. contents:: Contents
-   :local:
-   :depth: 1
+This article provides a quick overview and reference to how documentation is *hosted* on Read the Docs.
+Hosting is optimized with documentation in mind and a number of special features are provided,
+some of which may be customized.
+
+.. seealso::
+
+   :doc:`/reference/features`
+     For a full list of Read the Docs features, see the features reference page.
```

Comment:
```suggestion
   :doc:`Feature reference </reference/features>`
    A complete list of our features.
```

This feels more explicit and clearer? Probably should just rename this page to make it more obvious?

Reply:
I'll give it a quick rename :+1: The final destination can come later. I'm sure it's better to rename the actual page though than to give explicit labels in the `seealso::` boxes.